### ISSUE TITLE
[Bug Report] Radio group has accent color on startup if error is true

### ISSUE THREAD

**Issue by T9C5F** [vuetifyjs/vuetify#3599]
### Versions and Environment
**Vuetify:** 1.0.8
**Vue:** 2.5.16
**Browsers:** Chrome 65.0.3325.162
**OS:** Windows 10

### Steps to reproduce
```
<v-radio-group error>
   <v-radio label="Option"></v-radio>
</v-radio-group>
```

### Expected Behavior
Option should have error text color and error animation

### Actual Behavior
Option has the correct error animation but has the accent text color

### Reproduction Link
[https://codepen.io/t9c5f/pen/RMGeWy](https://codepen.io/t9c5f/pen/RMGeWy)

**Comment by praveenpuglia** [comment]
It seems like the radio inside doesn't know about the parent being in error state by the time it's classes are generated. 

Take a look at this - https://github.com/vuetifyjs/vuetify/blob/dev/src/components/VRadioGroup/VRadio.js#L54-L58

```js
if (!this.parentError) {
  return this.addTextColorClassChecks(classes)
}
return classes
```

Thoughts @KaelWD ?

**Comment by johnleider** [comment]
This has been resolved as of v1.1
